graspy/inference/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .latent_position_test import LatentPositionTest
 from .latent_distribution_test import LatentDistributionTest
+from .dists import *
```

Comment:
Don't use import *

Reply:
also are these things you want to be part of the public API? just wondering

Reply:
these are just example functions I can use for tests, as well as defaults. I'll remove euclidean and add is as a default when the string "euclidean" is passed and I'll add gaussian within LDT.